packages/ckeditor5-engine/src/view/downcastwriter.js

```diff
@@ -272,6 +273,36 @@ export default class DowncastWriter {
 		return uiElement;
 	}
 
+	/**
+	 * Creates a new {@link module:engine/view/rawelement~RawElement}.
```

Comment:
Does it make sense to create raw elements without their "internals" (passing the callback)?

Also, I'd focus on having here a good cross-ref between UIElement and RawElement, explaining also briefly what's the difference, etc. These API methods have a high chance of being visited and it's hard to figure out what it is based on this method.

Reply:
What about this? I can see that the callback is still optional.

I'll remove the first two examples and leave the rest as is to make it less confusing.

Reply:
I can see it's a copy-paste from UI element... But it makes more sense there as sometimes UIElements can be used without any content as their purpose is different.

Reply:
Oh, I remember now that you mentioned that it'd look bad if the last param wasn't optional since attrs are. So, that's fine. I'll just remove those examples.